src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/adaptiveForm/checkbox/.content.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:description="Adaptive Form Check Box (v1) component"
+    jcr:primaryType="cq:Component"
+    jcr:title="Check Box"
+    sling:resourceSuperType="core/fd/components/form/checkbox/v1/checkbox"
```

Comment:
@barshat7 do we also need to update the core components version in archetype before this change?

Reply:
this will be done once we have the final release for cc